server/.env

```diff
@@ -1 +1 @@
-TEAM_MEMBERS="Aecio, Bonnie, Shums Kassam"
+TEAM_MEMBERS="Aecio, Bonnie, Shums Kassam, Seshunath Kotra"
```

Comment:
If you can, reverting this commit may be a good idea to avoid conflicts with @uzher-code 's PR removing the env file

Reply:
Removed .env file in this branch.